data-prepper-plugins/service-map-stateful/src/main/java/com/amazon/dataprepper/plugins/prepper/ServiceMapStatefulPrepper.java

```diff
@@ -154,6 +170,7 @@ private static File createPath(File path) {
                             .map(serviceMapRelationship -> {
                                 try {
                                     relationshipState.add(serviceMapRelationship);
+                                    uniqueRelationshipsCounter.increment();
```

Comment:
This may be a little confusing for the end user, as this number will likely be much higher than the number of relationships they see in the service map plugin. We may need to think a bit more about how we want to calculate this.

Reply:
Looking into it